### ISSUE TITLE
OpenZimbra project

### ISSUE THREAD

**Issue by adriangibanelbtactic** [beezimOSS/beezoss#5]
I just found the [OpenZimbra project](https://github.com/OpenZimbra) which tries to work on:
* Porting packages to Debian
* Improve the build system so that you no longer need ./install.sh script but you can use Debian packages as usual.
* [zm-build modified branch for 8.8.10](https://github.com/OpenZimbra/zm-build/tree/mtx/maint-8.8.10)

Update: It seems that the OpenZimbra project was discarded and replaced/improved by [LibreZimbra project](https://github.com/LibreZimbra/librezimbra).

@metux it would be nice if you checked the current issues in the project and give us some of your insights.

**Comment by Polve** [comment]
The OpenZimbra project is invalid, maybe this is the right one?

https://github.com/LibreZimbra

**Comment by adriangibanelbtactic** [comment]
> The OpenZimbra project is invalid, maybe this is the right one?
> 
> https://github.com/LibreZimbra

That's right @Polve . It seems that @metux discarded its own base and started with the new Zimbra source code base.
I have updated my original post.

Thank you for your feedback.

**Comment by metux** [comment]
> The OpenZimbra project is invalid, maybe this is the right one?
> 
> https://github.com/LibreZimbra

Yes, I had renamed the org (OpenZimbra had been mine too, started aeons ago and fell asleep ;-))

LZ isn't just about getting Zimbra built somehow, but also refactoring the whole build process
(eg. now purely package based - each repo is built on its own).

And even more: it's about fully automated deployment and provisioning, including containerized deployments.
That's why I'm doing that one component after another. So far, got the LDAP nodes (with automatic cluster
setup) up and running and now taking care of mailboxd node.

--mtx
